--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   src/third_parties/rsl_rl_local/rsl_rl/algorithms/ppo.py

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/src/third_parties/rsl_rl_local/rsl_rl/algorithms/ppo.py b/src/third_parties/rsl_rl_local/rsl_rl/algorithms/ppo.py
index 915c20d..148556c 100644
--- a/src/third_parties/rsl_rl_local/rsl_rl/algorithms/ppo.py
+++ b/src/third_parties/rsl_rl_local/rsl_rl/algorithms/ppo.py
@@ -4,7 +4,6 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 from __future__ import annotations
-
 import torch
 import torch.nn as nn
 import torch.optim as optim